centralized_web_server/api.toml

```diff
@@ -0,0 +1,60 @@
+[meta]
+NAME = "hotshot_centralized_web_server"
+DESCRIPTION = "Centralized web server for HotShot"
+FORMAT_VERSION = "0.1.0"
+
+# GET the proposal for a view, where the view is passed as an argument
+[route.getproposal]
+PATH = ["proposal/:view_number"]
+":view_number" = "Integer"
+DOC = """
+Return the proposal for a given view number
+"""
+
+# POST a proposal, where the view is passed as an argument
+[route.postproposal]
+PATH = ["proposal/:view_number"]
+METHOD = "POST"
+":view_number" = "Integer"
+DOC = """
+Post the proposal for a given view_number
+"""
+
+# GET all the votes from a given index for a given view number
+[route.getvotes]
+PATH = ["votes/:view_number/:index"]
+":view_number" = "Integer"
+":index" = "Integer"
+METHOD = "GET"
+DOC = """
+Get all votes for a view number
+"""
+
+# POST a vote, where the view number is passed as an argument
+[route.postvote]
+PATH = ["votes/:view_number"]
+":view_number" = "Integer"
+METHOD = "POST"
+DOC = """
+Send a vote
+"""
+
+# GET all transactions starting at :index
+[route.gettransactions]
+PATH = ["transactions/:index"]
+":index" = "Integer"
+METHOD = "GET"
+DOC = """
+Get all transactions since given index
+"""
+
+
+# POST a transaction
+[route.posttransaction]
+PATH = ["transactions"]
+METHOD = "POST"
+DOC = """
+Send a transactions
```

Comment:
Nit: I'm a bit confused about whether we want to support sending multiple transactions. If so, maybe change "a transaction" in lines 52 and 57 to the plural form?

Reply:
I'll change it!  It the future we would support sending multiple transactions, but right now we only support sending one at a time.